frontend/scripts/openapi.yaml

```diff
@@ -976,6 +976,10 @@ paths:
         name: language
         schema:
           type: string
+      - in: query
+        name: uploader
+        schema:
+          type: string
```

Comment:
Has the schema been generated automatically? On this branch, the new `OpenApiParameter` has not been defined in [`views/video.py`](https://github.com/tournesol-app/tournesol/blob/c484c58d4a8c176c7d5dcdf1436086f040f0b5e3/backend/tournesol/views/video.py#L33-L60)

Reply:
Fixed